docker-compose.yml

```diff
@@ -7,6 +7,7 @@ volumes:
   panamax-io-index-vol:
   resource-recorder-vol:
   mongodb-vol:
+  logger-vol:
```

Comment:
We should remember to remove this when switching to Postgres

Reply:
I will add a note about this to the logger postgres ticket.